packages/@aws-cdk/aws-pipes-enrichments-alpha/test/lambda.test.ts

```diff
@@ -0,0 +1,90 @@
+import { DynamicInput, InputTransformation, Pipe } from '@aws-cdk/aws-pipes-alpha';
+import { App, Stack } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import { Code, Function, Runtime } from 'aws-cdk-lib/aws-lambda';
+import { TestSource, TestTarget } from './test-classes';
+import { LambdaEnrichment } from '../lib';
+
+describe('lambda', () => {
+  let lambda: Function;
+  let stack : Stack;
+  beforeEach(() => {
+    const app = new App();
+    stack = new Stack(app, 'TestStack');
+    const enrichmentHandlerCode = 'exports.handler = async (event) => { return event.map( record => ({...record, body: `${record.body}-${record.name}-${record.static}` }) ) };';
+    lambda = new Function(stack, 'EnrichmentLambda', {
+      code: Code.fromInline(enrichmentHandlerCode),
+      handler: 'index.handler',
+      runtime: Runtime.NODEJS_LATEST,
+    });
+  });
+
+  it('should have only enrichment arn', () => {
```

Comment:
I think using `it` is fine but elsewhere in CDK we have been using `test` and we may eventually want parity if/when we stabilize this module.